src/routing/AppRoutes.tsx

```diff
@@ -4,18 +4,21 @@ import { DashboardContainer } from 'app/dashboard/DashboardContainer';
 import { AppRoute } from './AppRoute.enum';
 import { PageNotFound } from 'ui';
 import { useLocale } from 'hooks';
+import { ProfileContainer } from 'app/profile/ProfileContainer';
 import { LoginContainer } from 'app/login/LoginContainer';
 
 export const AppRoutes = () => {
   const { formatMessage } = useLocale();
 
   return (
     <Routes>
-      <Route path={AppRoute.home} element={<DashboardContainer />} />
-      <Route path={AppRoute.login} element={<LoginContainer />} />
+      <Route path={AppRoute.HOME} element={<DashboardContainer />} />
+      <Route path={AppRoute.LOGIN} element={<LoginContainer />} />
+      <Route path={`${AppRoute.PROFILE}/:name`} element={<ProfileContainer />} />
```

Comment:
Teraz tak patrzę - właściwie to ten sufix `/:name` powinien być jednak w całości wewnątrz `AppRoute.PROFILE`, bo jest to część routa

Reply:
Tylko, że wtedy jak bym używał `AppRoute.PROFILE` w linku to by mi zwracał `http://localhost:3001/profile/:name/Marcin`